modules/display.c

```diff
@@ -365,6 +367,37 @@ static gboolean backlight_ioctl(int value)
 	return status;
 }
 
+/* Set DPMS mode for X11 display to on or off
+ *
+ * @params dpms_on To force the mode into on or off (blanked) mode.
+ */
+static void display_dpms(gboolean dpms_on)
```

Comment:
I'd rename: display_dpms() -> dpms_set_display_on()

Reply:
Changed to `display_dpms_set`

Reply:
What this function does is making the display turn on or off using DPMS.
The name of function display_dpms_set(TRUE) doesn't give a programmer the idea what the function purpose is. The programmer may think: does it enable DPMS?
So the programmer will be forced to check the sources of the function to get the idea what it does exactly.
I suggest to make the function names saying what is the purpose of a function, so other people will not have to guess about it.

Reply:
I think it does give clear hints about the function does. It sets DPMS on or off.

`dpms_set_display_on()` makes me think the function only turns it on, and it does not do that.

Reply:
See the context: `dpms_set_display_on(FALSE)`
Will you think that the function turns the display on with this call?

Reply:
I think the same is true for `display_dpms_set(FALSE)`. It in fact also mirrors what the utility `xset` does: `xset dpms force off`. `xset dpms force on`.

Reply:
The key word is "force" here.

Reply:
Compare to this options of xset:
```
       -dpms   The -dpms option disables DPMS (Energy Star) features.
       +dpms   The +dpms option enables DPMS (Energy Star) features.
```

Reply:
```
The option can take up to three numerical values, or  the
`force'  flag  followed  by  a  DPMS  state.  The `force' flags
forces the server to immediately switch to the DPMS state spec‐
ified.   The  DPMS  state  can  be one of `standby', `suspend',
`off', or `on'.  When numerical values are given, they set  the
inactivity  period (in units of seconds) before the three modes
are activated.
```

Reply:
Ok, I still don't agree, but I also don't care as much as you do, and I don't want either of us to waste more time on it, so I'll rename it to what you want. ;)

Reply:
Well, my variant seems a bit uggly... but it's more clear now what the function purpose is.
If we can find a better name, I'm fine to change to it :)